infra/resources/modules/commons/apim.tf

```diff
@@ -200,3 +208,23 @@ resource "azurerm_api_management_subscription" "test" {
   state               = "active"
   allow_tracing       = false
 }
+
+####################################################################################
+# IO Backend User
+####################################################################################
+resource "azurerm_api_management_user" "io_backend" {
+  user_id             = "io-backend"
+  api_management_name = module.apim.name
+  resource_group_name = module.apim.resource_group_name
+  first_name          = "IO"
+  last_name           = "Backend"
+  email               = "io-platform@pagopa.it"
```

Comment:
**note**: not sure this is the right email address

@AleDore what do you think? can we keep this or should we use a different address?

Reply:
this one can do the job :)